sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Model-Quality-With-SageMaker-Pipelines-On-Demand.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Please only use present tense (we will showcase -&gt; we showcase, we will create -&gt; we create). Make sure this applies to the rest of the notebook and other notebooks in this PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>From my review view, there is a double space between 'we' and 'create' - is that true for you?</p>

Reply:
<p>Just curious, can you see this comment?</p><p> </p>

Reply:
<p>Must address before merge!</p>

Reply:
ack

Reply:
done